pom.xml

```diff
@@ -52,9 +52,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
It's probably better to upload the new aws-sdk to the jenkins maven repo and get it from there